cardano-node-socket-emulator/src/Cardano/Node/Socket/Emulator/Chain.hs

```diff
@@ -66,17 +69,13 @@ fromEmulatorChainState EC.ChainState {EC._txPool, EC._index, EC._chainCurrentSlo
                       , _txPool      = _txPool
                       , _index       = _index
                       , _currentSlot = _chainCurrentSlot
-                      , _tip         = listToMaybe _chainNewestFirst
+                      , _tip         = O.TipGenesis
```

Comment:
Should it always be genesis?

Reply:
Oh good point! It was true in the way it was used, but not in general.

Reply:
Fixed. Can you merge this?